### ISSUE TITLE
Unable to control the template state for various sizes

### ISSUE THREAD

**Issue by isha-limdiwala** [polotno-project/polotno-studio#23]
I am having trouble controlling the template state for different sizes in the Polotno editor.

I'm initially adjusting the style or placements of items in the design and choosing the size of the Instagram post (1080x1080 pixels). I then switched to a different size IG story (1080x1920 pixels) and made any necessary style or positioning adjustments. In this instance, the changes I made are no longer available if I return to the size of an Instagram post (1080x1080 pixels). It has been replaced by the latest adjustments made to various sizes.

So, is there a way to have distinct states for templates depending on the available sizes? Therefore, if we last saved changes for a certain change, we don't need to make changes repeatedly with different sizes.

https://www.screencast.com/t/9dXCu2lqc2oq

**Comment by lavrton** [comment]
No, currently that is not supported. There is only one "current" state with its own size.

I think the solution will be to duplicate the page and set its own size. The current "resize" menu doesn't support that, but I think I can add that function.

**Comment by isha-limdiwala** [comment]
Thanks for your reply. So the function will work in different sizes per page. If you could include that feature, that would be fantastic. Any ETA on that feature as of now?
